cmd/sqlcmd/sqlcmd.go

```diff
@@ -262,66 +286,127 @@ func setFlags(rootCmd *cobra.Command, args *SQLCmdArguments) {
 	rootCmd.Flags().BoolVarP(&args.TrimSpaces, "trim-spaces", "W", false, localizer.Sprintf("Remove trailing spaces from a column"))
 	rootCmd.Flags().BoolVarP(&args.MultiSubnetFailover, "multi-subnet-failover", "M", false, localizer.Sprintf("Provided for backward compatibility. Sqlcmd always optimizes detection of the active replica of a SQL Failover Cluster"))
 
-	rootCmd.Flags().StringVarP(&args.Password, "password", "P", "", localizer.Sprintf("Obsolete. The initial passwords must be set using the %s environment variable or entered at the password prompt", localizer.PasswordEnvVar2))
+	rootCmd.Flags().StringVarP(&args.Password, "password", "P", "", localizer.Sprintf("Password"))
 
 	// Using PersistentFlags() for ErrorSeverityLevel due to data type uint8 , which is not supported in Flags()
 	rootCmd.PersistentFlags().Uint8VarP(&args.ErrorSeverityLevel, "error-severity-level", "V", 0, localizer.Sprintf("Controls the severity level that is used to set the %s variable on exit", localizer.ErrorLevel))
 
-	var screenWidth int
-	screenWidthPtr := &screenWidth
-	rootCmd.Flags().IntVarP(screenWidthPtr, "screen-width", "w", 0, localizer.Sprintf("Specifies the screen width for output"))
+	_ = rootCmd.Flags().IntP(screenWidth, "w", 0, localizer.Sprintf("Specifies the screen width for output"))
+	_ = rootCmd.Flags().IntP(variableTypeWidth, "y", 256, localizer.Sprintf("Sets the sqlcmd scripting variable %s", "SQLCMDMAXVARTYPEWIDTH"))
+	_ = rootCmd.Flags().IntP(fixedTypeWidth, "Y", 0, localizer.Sprintf("Sets the sqlcmd scripting variable %s", "SQLCMDMAXFIXEDTYPEWIDTH"))
```

Comment:
Can you check if you are getting any errors when generating localizations JSONs by running `go generate .\internal\translations\translations.go`
The gotext tool uses the string itself as a key when creating the map and if two keys are same then it simply throws warning and ignores the conflicting strings in resulting JSON file.

Following is the error for the above string
```
git\go-sqlcmd>go generate .\internal\translations\translations.go
Key "Sets the sqlcmd scripting variable %s" matches conflicting messages: [Sets the sqlcmd scripting variable {SQLCMDMAXVARTYPEWIDTH}] and [Sets the sqlcmd scripting variable {SQLCMDMAXFIXEDTYPEWIDTH}]
```

Reply:
is there a way to tell gotext to suppress the warnings about missing translations? It'd be ideal for `build\build` to only print the errors that can be fixed by code.

Reply:
Not that I know of.